projects/packages/connection-ui/src/class-admin.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * The Connection UI Admin Area.
+ *
+ * @package automattic/jetpack-connection-ui
+ */
+
+namespace Automattic\Jetpack\ConnectionUI;
+
+/**
+ * The Connection UI Admin Area
+ */
+class Admin {
+
+	/**
+	 * Construction.
+	 */
+	public function __construct() {
+		add_action( 'admin_menu', array( $this, 'register_submenu_page' ), 1000 );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+	}
+
+	/**
+	 * Initialize the UI.
+	 */
+	public static function init() {
+		add_action(
+			'plugins_loaded',
+			function () {
+				new static();
+			}
+		);
+	}
+
+	/**
+	 * Register's submenu.
+	 */
+	public function register_submenu_page() {
+		add_submenu_page(
+			'tools.php',
+			'Connection Manager',
+			'Connection Manager',
```

Comment:
You'll probably want those to be translatable:

```suggestion
			__( 'Connection Manager', 'jetpack' ),
			__( 'Connection Manager', 'jetpack' ),
```